src/main/scala/nl.knaw.dans.easy.authinfo/components/FileItem.scala

```diff
@@ -48,7 +51,9 @@ object FileItem {
     "easy_owner" -> "owner",
     "easy_visible_to" -> "visibleTo",
     "easy_accessible_to" -> "accessibleTo",
-    "easy_date_available" -> "dateAvailable"
+    "easy_date_available" -> "dateAvailable",
+    "easy_license_key" -> "licenseKey",
+    "easy_license_title" -> "licenseTitle"
```

Comment:
this causes the diff to be less confusing next time around 😉

```suggestion
    "easy_license_title" -> "licenseTitle",
```